content/en/tracing/faq/app_analytics_agent_configuration.md

```diff
@@ -4,8 +4,15 @@ kind: Documentation
 aliases:
   - /tracing/trace_search_and_analytics/agent_trace_search/
   - /tracing/app_analytics/agent_trace_search/
+  - /tracing/guide/app_analytics_agent_configuration/
 ---
 
+<div class="alert alert-danger">
+This is a deprecated page with configuration information relevant to Legacy App Analytics. Tracing without Limits™ is now the recommended way to have full control over your data ingestion and trace retention with no sampling.
+<br>
+Migrate to <a href="https://docs.datadoghq.com/tracing/trace_retention_and_ingestion">Trace Retention and Ingestion</a> to use the new functionality.
```

Comment:
I think telling people to "migrate" here is deceptive, since they don't have to do anything.
```suggestion

```

Reply:
For best results they'll drop DD_<INTEGRATION>_ANALYTICS_ENABLED env vars and set the new one DD_TRACE_SAMPLE_RATE=1.0